src/project.js

```diff
@@ -77,6 +78,29 @@ class Project extends Model {
     }
   }
 
+  // Layers the contents of an atomProject file's config
+  // on top of the current global config.
+  replace (newSettings) {
+    if (newSettings == null) {
+      atom.config.clearProjectSettings()
+      this.setPaths([])
+      this.projectSettings = null
+    } else {
+      atom.config.resetProjectSettings(newSettings.config)
+      this.projectSettings = newSettings
+      this.setPaths(newSettings.paths)
+    }
+    this.emitter.emit('did-replace', newSettings)
+  }
+
+  onDidReplace (callback) {
+    return this.emitter.on('did-replace', callback)
+  }
+
+  getProjectSettings () {
+    return this.projectSettings
+  }
```

Comment:
Why does the `Project` need to store the settings, as opposed to them just being stored in the config?

Reply:
This is just so that third party packages can see which project is active (as well as which settings, paths, etc are on the project). I can change the way this is handled though if you think this shouldn't live on project.

Reply:
i think the only thing we need here is the path to the file so that we can have UI to show the active project?

Reply:
Alright - then we can just grab it from config! (Or we can keep it on project, I'm not opinionated either way).